lib/aggregate.rb

```diff
@@ -23,4 +23,40 @@
 require "aggregate/container"
 
 module Aggregate
+  class << self
+    attr_writer :configuration
```

Comment:
Considering this will be mixed in to models, can we make the names more specific?